ocp_resources/data_import_cron.py

```diff
@@ -0,0 +1,53 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class DataImportCron(NamespacedResource):
+    """
+    DataImportCron object
+    """
+
+    api_group = NamespacedResource.ApiGroup.CDI_KUBEVIRT_IO
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        client=None,
+        source=None,
```

Comment:
This is no longer source but 
```Template cdiv1.DataVolume `json:"template"```

https://github.com/kubevirt/containerized-data-importer/blob/main/staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/types.go#L482